site/src/pages/components/invokers.explainer.mdx

```diff
@@ -326,6 +302,302 @@ Invokers, the names and exact semantics may be subject to change:
+journey is maintained.
+
+</details>
+
+<br/><br/>
+
+<details>
+<summary>Other considerations not explicitly proposed.</summary>
+
+##### aria-pressed
+
+Given elements will have `aria-expanded`, adding `aria-pressed` would be
+confusing or redundant, and as such won't be proposed for these buttons.
+
+##### aria-controls
+
+While `aria-controls` attempts to establish a similar style of relationship to
+`aria-details`, it is largely intented for parent-child relationships. Buttons
+that open popovers do not necessarily represent a parent-child relationship.
+
+</details>
+
+<br/><br/>
+
+#### Buttons with `command=show-modal`
+
+Buttons with this dialog commands will implicitly receive `aria-haspopup=dialog`.
+
+<details>
+<summary>Why?</summary>
+
+Some Assistive Technologies will announce that the button will open a dialog,
+for example `<button aria-haspopup=dialog>Delete</button>` may be announced in
+a screen reader as "Delete, button, opens dialog". This can be important
+information for a user as it allows them to anticipate a focus change or a
+change in the pages navigable elements which may otherwise not be quite as
+apparent.
+
+However, we may revisit this decision based on more evidence, research and
+discussion. Buttons that open dialogs rarely come with additional visual
+treatment, and so the lack of a visual analog may allude to the presence of
+`aria-haspopup=dialog` being redundant.
+
+This decision _may also_ be revisited in light of any new elements proposals
+that have the role of `menu`, `listbox`, `tree` or `grid`. It may be proposed
+that, when connected to those elements via `commandfor`, buttons will
+implicitly recieve `aria-haspopup` of the matching type.
+
+</details>
+
+<br/><br/>
+
+Buttons with this dialog command will also have focus restored to the button
+when the dialog closes. This will require the dialog keeping a reference to the
+opening button so that it can return the focus to that button.
+
+<details>
+<summary>Why?</summary>
+
+It is important for keyboard users (or other users who rely on focus) to not
+have their focus state lost when performing actions on the page. If the page
+shows a new piece of navigable UI, and shifts the user's focus to that UI,
+then it is important to restore focus when that UI closes, so that the user's
+journey is maintained.
+
+</details>
+
+<br/><br/>
+
+<details>
+<summary>Other considerations not explicitly proposed.</summary>
+
+##### aria-details
+
+Exposing an `aria-details` relationship between the button and the dialog is
+redundant. These relationships are useful only when the related element is
+open and present on the page and the invoking button is still navigable. A
+modal dialog renders the rest of the page inert meaning that it won't be
+possible to navigate to the invoking button.
+
+##### aria-pressed
+
+The `aria-pressed` state is the incorrect type of association for elements
```

Comment:
it's weird this says aria-expanded would be the correct state to expose, but in the next section also talks about how the state is redundant.